CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
+  - Before, we stored imported ansible config in ansible-imported-configs which is hidden for users, in this release, we merge the configs into meta.yaml so that the user can see the config with the command `tiup cluster edit`
+- Enhance the `tiup mirror` command ([#860](https://github.com/pingcap/tiup/pull/860), [@lucklove](https://github.com/lucklove))
+  - **Support merge two or more mirrors into one**
+  - Support publish component to local mirror besides remote mirror
+  - Support add component owner to local mirror
+- Support deploy cluster with hostname besides ip address (EXPERIMENTAL) ([#948](https://github.com/pingcap/tiup/pull/948),[#949](https://github.com/pingcap/tiup/pull/949), [@fln](https://github.com/fln))
+- Support setting custom timeout for waiting instances up in tiup-playground ([#968](https://github.com/pingcap/tiup/pull/968), [@unbyte](https://github.com/unbyte))
+- Support check and disable THP in tiup cluster check ([#964](https://github.com/pingcap/tiup/pull/964), [@anywhy](https://github.com/anywhy))
+- Support sign remote manifest and rotate root.json ([#967](https://github.com/pingcap/tiup/pull/967), [@lucklove](https://github.com/lucklove))
+
+### Fixes
+
+- Fix the issue that public key created by TiUP not removed after the cluster destroyed ([#910](https://github.com/pingcap/tiup/pull/910), [@9547](https://github.com/9547))
+- Fix the issue that user defined username and password not imported from tidb-ansible cluster correctly ([#937](https://github.com/pingcap/tiup/pull/937), [@AstroProfundis](https://github.com/AstroProfundis))
+- Fix the issue that tiup-playground not quit components with correct order: TiDB -> TiKV -> PD ([#933](https://github.com/pingcap/tiup/pull/933), [@unbyte](https://github.com/unbyte))
+- Fix the issue that tikv reports wrong advertise address when `--status-addr` is set to a wildcard address like `0.0.0.0` ([#951](https://github.com/pingcap/tiup/pull/951), [@lucklove](https://github.com/lucklove))
+- Make error message of monitor port conflict more readable ([#966](https://github.com/pingcap/tiup/pull/966), [@JaySon-Huang](https://github.com/JaySon-Huang))
+- Fix the issue that the cluster can't start when there are multiple prometheus ([#972](https://github.com/pingcap/tiup/pull/972), [@9547](https://github.com/9547))
+- Fix the issue that prometheus doesn't reload target after scale-in action ([#958](https://github.com/pingcap/tiup/pull/958), [@9547](https://github.com/9547))
+- Fix the issue that the config file for tiflash missing in tiup-palyground ([#969](https://github.com/pingcap/tiup/pull/969), [@unbyte](https://github.com/unbyte))
+- Fix the issue that the tiflash doesn't create err log while numactl is missing ([#984](https://github.com/pingcap/tiup/pull/984), [@lucklove](https://github.com/lucklove))
```

Comment:
- Fix Tilfash startup failure without stderr output When numa is enabled but numactl cannot be found ([#984](https://github.com/pingcap/tiup/pull/984), [@lucklove](https://github.com/lucklove))